Z/ZlibNG/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ZlibNG"
+version = v"2.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/zlib-ng/zlib-ng.git", "b3dcf11b4204a16fde71fa8224d7b7054e225b93")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/zlib-ng
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_C_FLAGS="-O3" \
+    ..
+make -j${nproc}
+make install
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms(exclude=[Platform("x86_64", "freebsd")])
```

Comment:
What's the problem with FreeBSD?

Reply:
`error: use of undeclared identifier 'isascii'` at various places

[freebsd_error.txt](https://github.com/JuliaPackaging/Yggdrasil/files/9938063/freebsd_error.txt)

Reply:
https://www.freebsd.org/cgi/man.cgi?query=isascii&sektion=3&apropos=0&manpath=freebsd

Reply:
The error comes from a test only dependency `gtest.h` so I think disabling tests would get rid of the problem... Probably the reason why https://github.com/JuliaPackaging/Yggdrasil/pull/3614 disabled them as well. Or is that a nono?:)

Reply:
We don't run tests, so removing them is them.